geo/src/utils.rs

```diff
@@ -53,3 +53,21 @@ where
         }
     }
 }
+
+// The Rust standard library has `max` for `Ord`, but not for `PartialOrd`
+pub fn partial_max<T: PartialOrd>(a: T, b: T) -> T {
+    if a > b {
+        a
+    } else {
+        b
+    }
+}
+
+// The Rust standard library has `min` for `Ord`, but not for `PartialOrd`
+pub fn partial_min<T: PartialOrd>(a: T, b: T) -> T {
+    if a > b {
```

Comment:
shouldn't this be `<` since this method wants the min?

Reply:
yep totally, good catch! addressed in https://github.com/georust/geo/pull/443/commits/ff08c8aa044fb8f9c9b1e545977d396539a9875e